implementations/rust/ockam/ockam_transport_tcp/src/workers/receiver.rs

```diff
@@ -83,6 +83,7 @@ impl TcpRecvProcessor {
         );
         ProcessorBuilder::new(receiver)
             .with_mailboxes(Mailboxes::new(mailbox, vec![internal]))
+            .terminal(addresses.receiver_address().clone())
```

Comment:
Might also put TCP attribute

Reply:
The same reasoning of identity applies here as well, decoupling transport protocol